services/galley/test/integration/API/Teams/LegalHold.hs

```diff
@@ -18,6 +17,7 @@
 --
 -- You should have received a copy of the GNU Affero General Public License along
 -- with this program. If not, see <https://www.gnu.org/licenses/>.
+{-# OPTIONS_GHC -Wno-unused-top-binds #-}
```

Comment:
We can avoid this by wrapping the test lines 156, 157 below into `when False` rather than removing them completely.  But not now.